src/main/java/com/example/TicketBookingApp/exception/ReservationNotFoundException.java

```diff
@@ -0,0 +1,5 @@
+package com.example.TicketBookingApp.exception;
+
+public class ReservationNotFoundException extends RuntimeException{
```

Comment:
a to nie powinno się mapować na kod 404 w odpowiedziach HTTP??

Reply:
Tak, mapowanie byłoby lepszym rozwiązaniem.